### PR TITLE
Exposing the underlying websocket in threadsafewebsocket

### PR DESCRIPTION
**Pull request by DotNetRussell**
## Proposed Changes

Please allow access to the underlying web socket 

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: src/Hopac.Websockets/Hopac.Websockets.fs [hunk 1]
```diff
@@ -214,12 +215,15 @@ module ThreadSafeWebSocket =
...
 
             let self = {
+                websocket = webSocket
                 sendCh = Ch ()
                 receiveCh = Ch ()
                 closeCh = Ch ()
                 closeOutputCh = Ch ()
             }
 
+            let webSocket = self.websocket
```
Comment (TheAngryByrd) [inline src/Hopac.Websockets/Hopac.Websockets.fs]: whats the need for these two lines?

Comment (DotNetRussell) [reply src/Hopac.Websockets/Hopac.Websockets.fs]: To allow access to it to an outside caller.

Comment (TheAngryByrd) [reply src/Hopac.Websockets/Hopac.Websockets.fs]: How does this allow access? You're already putting it in the record type.

Comment (DotNetRussell) [reply src/Hopac.Websockets/Hopac.Websockets.fs]: I thought that you needed to do this to allow access to it. I'll fix it

**Pull request merged**